triples-generator/src/test/scala/io/renku/triplesgenerator/events/categories/triplesgenerated/projectinfo/MemberEmailFinderSpec.scala

```diff
@@ -350,63 +352,26 @@ class MemberEmailFinderSpec
     val member  = projectMembersNoEmail.generateOne
 
     private implicit val logger: TestLogger[IO] = TestLogger[IO]()
-    val gitLabUrl          = GitLabUrl(externalServiceBaseUrl).apiV4
-    val commitAuthorFinder = mock[CommitAuthorFinder[IO]]
-    val finder = new MemberEmailFinderImpl[IO](commitAuthorFinder,
-                                               gitLabUrl,
-                                               Throttler.noThrottling,
-                                               retryInterval = 100 millis,
-                                               maxRetries = 1,
-                                               requestTimeoutOverride = Some(requestTimeout)
-    )
-  }
+    val gitLabUrl           = GitLabUrl(externalServiceBaseUrl).apiV4
```

Comment:
Is the `gitLabUrl` val used anywhere here now?
Similarly, I believe we should be able to take out the mixin to the `ExternalServiceStubbing` on this test class.